src/lib/updater.js

```diff
@@ -27,9 +27,7 @@ const packageInfo = require("../../package.json");
 class Updater {
   constructor() {
     this.cache = {};
-    this.updateUrl = `https://devices.ubuntu-touch.io/installer/${
-      packageInfo.package ? "?package=" + packageInfo.package : ""
-    }`;
+    this.updateUrl = `https://devices.ubuntu-touch.io/installer/`;
```

Comment:
no. the normal package links like https://devices.ubuntu-touch.io/installer?package=dmg still work and are crucial for usability. what has been broken on the websites is just the source package, but they just have to fix that. i'm not ready to accept a workaround for that here. filed a bug upstream: https://gitlab.com/ubports/infrastructure/devices.ubuntu-touch.io/-/issues/57

```suggestion
    this.updateUrl = `https://devices.ubuntu-touch.io/installer/${
      packageInfo.package ? "?package=" + packageInfo.package : ""
    }`;
```

Reply:
Great, thanks a lot for clarifying!